NSS-Day-Cohort-53/tabloid-full-stack-always-polite-iguanas
# view all categories

#### Changes Made
1. Added file `CategoryController.cs` to `Controller` directory.
2. Added file `Category.cs` to `Models`
3. Added files `CategoryRepository.cs` and `ICategoryRepository` to `repositories`
4. Added file `categoryList.js` to `src/components/Categories`
5. Added file `categoryManager.js` to `src/modules`
1. Modified file `Header.js` to include navigation links to category page for logged in users.
2. Modified file `ApplicationViews.js` to include routes for category list module
​
#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout 39_view_all_categories
    ```
6. Open a new Terminal tab (⌘T) and navigate to the server directory.
7. Test app functionality.
    > When a logged in user clicks on the Category Management link they are taken to a list of categories and each entry in the list has an "edit" and "delete" link option.
8. View code file.
    > Confirm file modifications are present as indicated above.
    > Confirm no unused code or extraneous comments exist.
